listenbrainz/listen.py

```diff
@@ -179,6 +179,8 @@ def to_influx(self, measurement):
             'time' : self.ts_since_epoch,
             'tags' : {
                 'user_name' : escape(self.user_name),
+                'artist_msid' : self.artist_msid,
```

Comment:
influx cannot store two rows with the same tags and time, so if we want tracks with the same timestamp but different metadata to be stored, we have to add `artist_msid` and `recording_msid` to tags as well. Otherwise the old listen will just get overwritten by the new one.